### DISCUSSION TITLE
Monthly Maintainers' Committee Meeting

### DISCUSSION THREAD

**Discussion by rprouse** [dotnet-foundation/Home#73] [Maintainers Committee]
The Maintainer's Committee meeting is set for the second Wednesday of every month at 16:00 EST, 19:00 UTC.  All are welcome to join.

The meeting invite `ICS` is in the attached `ZIP` file. Sorry, GitHub doesn't support iCal files 🤷🏻‍♂️ 

[Maintainers Committee Meeting.zip](https://github.com/dotnet-foundation/Home/files/7523012/Maintainers.Committee.Meeting.zip)

**Comment by barnson** [comment]
@rprouse Is the meeting open to lurkers?

**Reply by rprouse** [reply]
As far as I'm concerned, the meeting is open to everyone in our community. For those that can't make the meetings, I am looking for a way to post recordings now. They're a bit big for GitHub 😄

**Reply by barnson** [reply]
Thanks. Rob records the WiX meetings and posts them on YouTube: https://www.youtube.com/channel/UCVJeW1mjSujNPfsCQhsg7qQ. Not sure how well that works with Teams...

**Reply by rprouse** [reply]
Yes, we're looking at YouTube or public links from the Foundation Sharepoint site. I have the recording, it is just sharing a 455MB file 😄

**Reply by CalvinAllen** [reply]
@rprouse I don't think I have access to the recording, but if you need help acquiring the video / uploading / etc., let me know.  That seems like a bunch of administrative stuff you can hand off to somebody :)

**Reply by rprouse** [reply]
It automatically gets dropped in Sharepoint, so it should be as easy as sharing the link. We just need to adjust the Foundation's settings to allow external sharing.

**Comment by michael-hawker** [comment]
FYI I had to manually import the file vs. just being able to open the .ics directly... no idea why, but if someone else has the same issue:

1. In Outlook, select File > Open & Export > Import/Export.
2. In Import and Export Wizard box, select Import an iCalendar (.ics) or vCalendar file (.vcs) ,and then Next.

**Comment by michael-hawker** [comment]
Was the February meeting cancelled? I've been sitting in the lobby on the Teams call for about 5 minutes. Is there a monthly agenda that gets added to and posted somewhere?

**Reply by CalvinAllen** [reply]
Meeting for this month was moved to next Wednesday -
https://github.com/dotnet-foundation/wg-maintainers/issues/51

**Reply by michael-hawker** [reply]
Thanks for the info! @rprouse can you edit your initial post here to link to the other repo which posts the agendas and stuff? Thanks!

**Reply by rprouse** [reply]
Announcements and other info are in https://github.com/dotnet-foundation/wg-maintainers and in the maintainers channel in Slack.

**Reply by michael-hawker** [reply]
Thanks @rprouse, can you edit your initial post here with that link as well? It'll make it easier for others to discover when finding this discussion from the main hub here.

I also didn't realize the Slack was still active. I thought everything was moved to the Discord server https://aka.ms/dotnet-discord.

**Reply by rprouse** [reply]
That is the .NET Slack, I'm talking about the .NET Foundation Slack. Too many communication channels! https://join.slack.com/t/netfoundation/shared_invite/zt-c7np4gr7-rN5cjBt4iNFOgUE9seOfzA
